plugins/rocksdb/src/test/java/org/hyperledger/besu/plugin/services/storage/rocksdb/RocksDBKeyValuePrivacyStorageFactoryTest.java

```diff
@@ -48,15 +48,15 @@ public class RocksDBKeyValuePrivacyStorageFactoryTest {
   private final List<SegmentIdentifier> segments = List.of(TestSegment.DEFAULT, segment);
 
   @Test
-  public void shouldDetectVersion1DatabaseIfNoMetadataFileFound() throws Exception {
+  public void shouldDetectVersion1MetadataIfPresent() throws Exception {
     final Path tempDataDir = temporaryFolder.resolve("data");
     final Path tempDatabaseDir = temporaryFolder.resolve("db");
     final Path tempPrivateDatabaseDir = tempDatabaseDir.resolve("private");
     Files.createDirectories(tempPrivateDatabaseDir);
     Files.createDirectories(tempDataDir);
-    when(commonConfiguration.getStoragePath()).thenReturn(tempDatabaseDir);
-    when(commonConfiguration.getDataPath()).thenReturn(tempDataDir);
-    when(commonConfiguration.getDatabaseVersion()).thenReturn(DEFAULT_VERSION);
+    mockCommonConfiguration(tempDataDir, tempDatabaseDir);
+
+    Utils.createDatabaseMetadataV1Privacy(tempDataDir, 1, 1);
```

Comment:
For context and readability we should keep the removed lines 40 and 41 and use them here rather than the numbers itself

Reply:
good point, reused already present constants